src/Compilers/CSharp/Test/Syntax/Parsing/ExpressionParsingTests.cs

```diff
@@ -3533,5 +3533,127 @@ void M()
             EOF();
         }
 
+        [Fact, WorkItem(17683, "https://github.com/dotnet/roslyn/issues/17683")]
+        public void Bug17683a()
+        {
+            var source =
+@"from t in e
+where
+t == Int32.
+MinValue
+select t";
+            UsingExpression(source);
```

Comment:
The name of the helper doesn't really help the reader know that, though. I'll look at improving that in the future.